pdc_client/__init__.py

```diff
@@ -216,6 +233,8 @@ def decode(req):
                 token = self.obtain_token()
             self.session.headers["Authorization"] = "Token %s" % token
 
+        self._initialized = True
+
```

Comment:
I think we don't need to set the base class to _SetAttributeWrapper in PDCClient.

As all the attrs or items we get from PDCClient are now from Beanbag, so after that we just need to wrap the set function on _BeanBagWrapper.

What's your thinking?

Reply:
Method `__setattr__()` needs to be overridden in `PDCClient` to handle code like `client.resource = {...}` correctly.

Reply:
Make sense, if people set the resource url as server, like v1/releases/, the setattr should start from client itself.